src/component/playground/ContextBanner/ContextBanner.tsx

```diff
@@ -0,0 +1,39 @@
+import { colors } from '../../../themes/colors';
+
+interface Props {
+    context: object;
+}
+
+const styles = {
```

Comment:
Should use styled components here. We're moving away from defining styles in this way.

Reply:
Done